spec/spec.yml

```diff
@@ -4847,6 +4914,31 @@ definitions:
           blueBand:
             type: integer
 
+  DatasourceToCreate:
```

Comment:
Why not throw `DatasourceBands` into `DatasourceCreated` (which I think should be `DatasourceCreate`, for consistency)? Is there an advantage to a third object type for the flow from `POST` body -> a fully filled in datasource in the db?

Reply:
Per offline discussions, `bands` are either `{}` or `[{}, {}, ...]` in the return. Unless we upgrade to 3.0 (so that we could use something like `oneOf`, `anyOf`, ect.), we probably can't find a better way to deal with the above mentioned issue in 2.0 (which bravado supports).

Mark as resolved.